src/metabolic_ninja/resources.py

```diff
@@ -195,4 +354,7 @@ def register(path, resource):
     docs = FlaskApiSpec(app)
     register("/predictions", PredictionJobsResource)
     register("/predictions/<string:job_id>", PredictionJobResource)
+    register(
+        "/predictions/<string:job_id>/<string:row_id>", JobRowResource
```

Comment:
I think the term "row" is a bit generic. Prediction is probably better since it's already used. Both the URL and the other resource names are also very confusing to me.